pages/common/routers/form.js

```diff
@@ -81,14 +63,10 @@ module.exports = ({
   };
 
   const _checkChanged = (req, res, next) => {
-    getDefaultValues(req)
-      .then(defaultValues => {
-        if (some(req.form.values, (value, key) => hasChanged(value, key, defaultValues))) {
-          return next();
-        }
-        return next({ validation: { form: 'unchanged' } });
-      })
-      .catch(next);
+    if (some(req.form.values, (value, key) => hasChanged(value, key, req.model))) {
+      return next();
+    }
+    return next({ validation: { form: 'unchanged' } });
```

Comment:
I think this is something we'll only want for edit forms. An unchanged empty form should throw the field level validation messages.

If this happens _after_ the field-level validation is applied then we're 👌 but I wanted to check.